state-chain/pallets/cf-environment/src/lib.rs

```diff
@@ -520,6 +544,38 @@ pub mod pallet {
 			Ok(())
 		}
 
+		/// Manually initiates Assethub vault key rotation completion steps so Epoch rotation can be
+		/// continued and sets the Assethub Pure Proxy Vault in environment pallet. The extrinsic
+		/// takes in the hub_pure_proxy_vault_key, which is obtained from the Assethub blockchain as
+		/// a result of creating an assethub vault which is done by executing the extrinsic
+		/// create_assethub_vault(), hub_witnessed_aggkey, the aggkey which initiated the assethub
+		/// creation transaction and the tx hash and block number of the Assethub block the
+		/// vault creation transaction was witnessed in. This extrinsic should complete the Assethub
+		/// initiation process and the vault should rotate successfully.
+		///
+		/// ## Events
+		///
+		/// - [AssethubVaultCreationCallInitiated](Event::AssethubVaultCreationCallInitiated)
+		#[pallet::call_index(8)]
+		#[pallet::weight(Weight::zero())]
+		pub fn dispatch_solana_gov_call(
```

Comment:
Looks like the doc comments for this and the below call got mixed up (merge issue maybe?).